cfgov/v1/migrations/0006_contactsnippet_contactmolecules.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import wagtail.wagtailimages.blocks
+import wagtail.wagtailcore.fields
+import wagtail.wagtailcore.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('v1', '0005_eventarchivepage_eventrequestspeakerpage'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Contact',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('heading', models.CharField(max_length=22)),
+                ('body', wagtail.wagtailcore.fields.RichTextField(blank=True)),
+                ('contact_info', wagtail.wagtailcore.fields.StreamField(
+                    [
+                        (b'email', wagtail.wagtailcore.blocks.StructBlock(
+                            [
+                                (b'emails', wagtail.wagtailcore.blocks.ListBlock(wagtail.wagtailcore.blocks.StructBlock(
+                                        [
+                                            (b'label', wagtail.wagtailcore.blocks.CharBlock(max_length=22)),
+                                            (b'href', wagtail.wagtailcore.blocks.CharBlock(default=b'/'))
+                                        ], label=b'Email')))
+                            ])
+                        ),
+                        (b'phone', wagtail.wagtailcore.blocks.StructBlock(
+                            [
+                                (b'fax', wagtail.wagtailcore.blocks.BooleanBlock(default=False, required=False, label=b'Fax?')),
```

Comment:
Can this be broken into multiple lines to reduce horizontal scrolling in review?